packages/react-client/src/ReactFlightClient.js

```diff
@@ -1364,10 +1364,8 @@ function parseModelString(
           // happened.
           Object.defineProperty(parentObject, key, {
             get: function () {
-              // We intentionally don't throw an error object here because it looks better
-              // without the stack in the console which isn't useful anyway.
-              // eslint-disable-next-line no-throw-literal
-              throw (
+              // TODO: We should ideally throw here to indicate a difference.
+              return (
```

Comment:
This is a workaround because we read `model[0]` in two places in our own parsing which triggers this throw.